eng/build.sh

```diff
@@ -205,21 +205,35 @@ function BuildSolution {
     quiet_restore=true
   fi
 
+  coreclr_target_framework=netcoreapp2.0
+
   # build bootstrap tools
   bootstrap_config=Proto
   MSBuild "$repo_root/src/buildtools/buildtools.proj" \
     /restore \
+    /v:$verbosity \
     /p:Configuration=$bootstrap_config \
     /t:Build
 
   bootstrap_dir=$artifacts_dir/Bootstrap
-  mkdir -p "$bootstrap_dir"
-  cp $artifacts_dir/bin/fslex/$bootstrap_config/netcoreapp2.0/* $bootstrap_dir
-  cp $artifacts_dir/bin/fsyacc/$bootstrap_config/netcoreapp2.0/* $bootstrap_dir
+  if [ ! -d "$bootstrap_dir" ]; then
```

Comment:
@bretttfo We currently run out of the `netcoreapp2.1` folder even though we are doing this copying.  However the copying and directory existence test are useful to avoid endlessly rebuilding Proto.

Do you think we should run out of `Bootstrap` like on Windows?  I guess it doesn't matter for now.  Perhaps we don't really need to do it for Windows either

Reply:
I think running out of bootstrap is safer in case VS has FSharp.Build.dll locked

Reply:
Ah I see ok